.github/workflows/release-build.yml

```diff
@@ -20,19 +20,11 @@ jobs:
         with:
           job-id: jdk11
           remote-build-cache-proxy-enabled: false
-          arguments: updateVersionInDocs build --stacktrace -Prelease.version=${{ github.event.inputs.version }}
-      - name: Setup git name
-        run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
-      - name: Commit README updates
-        run: git commit -am "Releasing ${{ github.event.inputs.version }}"
+          arguments: build --stacktrace -Prelease.version=${{ github.event.inputs.version }}
       - name: Publish artifacts
         run: ./gradlew final --stacktrace -Prelease.version=${{ github.event.inputs.version }}
         env:
           BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
           BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
           GRGIT_USER: ${{ github.actor }}
           GRGIT_PASS: ${{ secrets.GITHUB_TOKEN }}
-      - name: Push README updates
-        run: git push
```

Comment:
are we missing the pushing of the tag to the main branch in here somewhere?

Reply:
`./gradlew final` will push it as part of the nebula release plugin. But fixed a mistake in my README :)